### DISCUSSION TITLE
Missing notes?

### DISCUSSION THREAD

**Discussion by colejohnson66** [dotnet/csharplang#8091] [LDM Notes]
The last meeting notes present in the repository are from April 8. Since then, supposedly five meetings have taken place (six if you count today), but their notes are missing. The ones for 15 Apr and 17 Apr are sitting as pull requests, so you could argue that only three (or four) are missing.

Side note, the description of this category has a typo: "meeting _nodes_"
![image](https://github.com/dotnet/csharplang/assets/11381599/daf40ea9-353c-4ebb-8663-bb82c18040de)

**Comment by 333fred** [comment]
I wouldn't count today's, since it hasn't taken place yet :slightly_smiling_face:. Monday's meeting will also not have published notes; it was an internal "meeting on the meeting", where we discussed the process of our meetings in general and make sure that everyone is still comfortable with how we run meetings. @KathleenDollard is working on notes for the other 2, since I was on vacation for those meetings.
